backend/store/migration/dev/LATEST.sql

```diff
@@ -47,7 +47,9 @@ CREATE TABLE principal (
     email TEXT NOT NULL,
     password_hash TEXT NOT NULL,
     idp_id INTEGER REFERENCES idp (id),
-    idp_user_info JSONB NOT NULL DEFAULT '{}'
+    idp_user_info JSONB NOT NULL DEFAULT '{}',
+    mfa_config JSONB NOT NULL DEFAULT '{}',
+    recovery_codes TEXT ARRAY NOT NULL DEFAULT '{}'
```

Comment:
Might be easier if we use TEXT without array, and let caller manipulate the string as a protojson.

Reply:
Move it into `mfa_config`